### DISCUSSION TITLE
Boilerplate Update

### DISCUSSION THREAD

**Discussion by BanDroid** [vikejs/vike#972] [Discussion] [answered]
will the boilerplate updated? having more ui library/framework as boilerplate. and could the boilerplate have setup ready for dynamic document props? hopefully the boilerplate could setup the code by default, because the documentation site and the examples were not very clear on the details, especially people still not familiar on this plugin

**Comment by brillout** [comment, marked as answer]
https://batijs.github.io/
https://github.com/brillout/vike-react
https://github.com/magne4000/vike-solid

Dynamic props aren't implemented by the `vike-*` integrators yet, but will be soon.

**Reply by BanDroid** [reply]
thank you, atleast i know in the future vike boilerplate would implement the dynamic props by default, setting up myself cause headache cause in dev works but prod not. it seems i implement it the wrong way, even though i copy-paste the examples
